appinventor/blocklyeditor/src/blocks/components.js

```diff
@@ -572,18 +602,6 @@ Blockly.Blocks.component_event = {
       if (varList.length != params.length) {
         return false; // parameters have changed
       }
-      for (var x = 0; x < varList.length; ++x) {
-        var found = false;
-        for (var i = 0, param; param = params[i]; ++i) {
-          if (componentDb.getInternationalizedParameterName(param.name) == varList[x]) {
```

Comment:
It seems like this causes the internationalization of parameter names to be dropped.

Reply:
I thought that it was just verifying that all of the parameter names matched the defaults, but maybe I misunderstood.  I don't see how that could happen in ordinary App Inventor programs, so it seemed safe to drop it, especially since it interferes with the purpose of this PR.

Reply:
Based on my review of the logs, I think it was to do with extension upgrading, i.e., if the parameter names changed then the block was likely no longer valid (or at least should be inspected). Prior to extensions, this code was not here.

Reply:
So, do you think it's worth keeping that check if it prevents Aptly users (including Codex) from using the non-default event parameter names?  Or maybe you know of a way to only do this check for extensions.

Reply:
As we discussed, I've adjusted the code to only do the check for extensions.